5.5/run-mysqld-slave.sh

```diff
@@ -6,6 +6,7 @@ source ${HOME}/common.sh
 set -eu
 
 export MYSQL_RUNNING_AS_SLAVE=1
+export MYSQL_MASTER_ADDR=${MYSQL_MASTER_ADDR:-mysql-master}
```

Comment:
OK, you can use https://github.com/kubernetes/kubernetes/blob/871ca7a05f3aeb0f2324b1d424ae67ada8425486/docs/user-guide/downward-api.md to get the current namespace.